tests/test_on_data/test_gin_ecephys/test_lfp.py

```diff
@@ -69,7 +87,10 @@ class TestConverter(NWBConverter):
         recording = converter.data_interface_objects["TestLFP"].recording_extractor
         with NWBHDF5IO(path=nwbfile_path, mode="r") as io:
             nwbfile = io.read()
-            nwb_lfp_electrical_series = nwbfile.processing["ecephys"]["LFP"]["ElectricalSeriesLFP"]
+            if expected_write_module == "raw":
+                nwb_lfp_electrical_series = nwbfile.acquisition["ElectricalSeriesLFP"]
+            else:
+                nwb_lfp_electrical_series = nwbfile.processing["ecephys"]["LFP"]["ElectricalSeriesLFP"]
             nwb_lfp_unscaled = nwb_lfp_electrical_series.data[:]
             nwb_lfp_conversion = nwb_lfp_electrical_series.conversion
```

Comment:
Do you think it might make sense to switch to an object-oriented test structure where we have an abstract Interface test and all the other interfaces inherit from it? That would make it easier to have specialized behavior for each interface, would allow us and would allow us to separate the interface tests into individual files that could handle testing the data I/O as well as the extraction of metadata.

Reply:
I know that would be a decent amount of refactoring work, but I think increasing the modularity there would allow us to scale more easily. For instance, a DataInterface being contributed from an external library would be able to have a test that inherits from the InterfaceTest class.

Reply:
obviously, this would be in a separate PR

Reply:
I'll respond to that idea on the corresponding issue